packages/codemods/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@suspensive/codemods",
+  "version": "0.1.0",
+  "description": "Codemods for @suspensive.",
+  "keywords": [
+    "suspensive",
+    "codemods"
+  ],
+  "homepage": "https://suspensive.org",
+  "bugs": "https://github.com/toss/suspensive/issues",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/toss/suspensive.git",
+    "directory": "packages/codemods"
+  },
+  "license": "MIT",
+  "author": "Gwansik Kim <gwansik.kim@gwansik.dev> & Jonghyeon Ko <jonghyeon@toss.im>",
+  "sideEffects": false,
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsup",
+    "ci:attw": "attw --pack",
+    "ci:eslint": "eslint \"**/*.{ts,tsx,cts,mts}\"",
+    "ci:publint": "publint --strict",
+    "ci:test": "vitest run --coverage --typecheck",
+    "ci:type": "tsc --noEmit",
+    "clean": "rimraf ./dist && rimraf ./coverage",
+    "prepack": "pnpm build",
+    "test:ui": "vitest --ui --coverage --typecheck"
+  },
+  "bin": "dist/bin/codemods.cjs",
+  "dependencies": {
+    "@commander-js/extra-typings": "^12.1.0",
+    "commander": "^12.1.0",
+    "execa": "^5.1.1",
```

Comment:
The latest version of execa is version 9, but we’re using version 5 in the codemods. Since version 6 and above only support ESM, we’ve opted to use version 5.